build.sbt

```diff
@@ -55,7 +55,7 @@ lazy val it = project
   .settings(
     DefaultBuildSettings.itSettings(),
     libraryDependencies ++= AppDependencies.it,
-    scalacOptions := (scalacOptions in ThisBuild).value.distinct
+    scalacOptions := (ThisBuild / scalacOptions).value.distinct
```

Comment:
Do we need to be setting `scalacOptions` explicitly? I'm not super familiar with this stuff, but I think this is just setting the options to the project default, which is the same as they'd be anyway.

Same with the setting on line 61 below.

If it's necessary for the `.distinct` call, that's _probably_ because we're manipulating the scalac options directly with `scalacOptions += "-Wconf:src=routes/.*:s",` above. That should be added via the tpolecat API.